source/_components/media_player.pioneer.markdown

```diff
@@ -23,10 +23,12 @@ To add a Pioneer receiver to your installation, add the following to your `confi
 media_player:
   - platform: pioneer
     host: 192.168.0.10
+    port: 23
 ```
 
 Configuration variables:
 
 - **host** (*Required*): The IP of the Pioneer device,  eg. `192.168.0.10`
 - **name** (*Optional*): The name you would like to give to the receiver.
+- **port** (*Optional*): The port on which the Pioneer device listens, e.g. `23`(default) or `8102`
```

Comment:
Would be good to mention here that some devices are using 8102 to communicate instead of the [source code](https://github.com/home-assistant/home-assistant/pull/3878/files#diff-7240066205c596fa0760f415052ff249R22).

Seems like that after 23 is a space missing.